src/WingetCreateCLI/Commands/BaseCommand.cs

```diff
@@ -431,17 +430,17 @@ protected async Task<bool> CheckGitHubTokenAndSetClient()
                 if (e is AuthorizationException)
                 {
                     Logger.ErrorLocalized(nameof(Resources.InvalidGitHubToken_Message));
+                    return false;
                 }
                 else if (e is RateLimitExceededException)
                 {
                     Logger.ErrorLocalized(nameof(Resources.RateLimitExceeded_Message));
+                    return false;
                 }
                 else if (e is NotFoundException)
                 {
                     Logger.ErrorLocalized(nameof(Resources.RepositoryNotFound_Error), this.WingetRepoOwner, this.WingetRepo);
                 }
-
-                return false;
```

Comment:
Instead of changing this to succeed on rep not found, why don't we just not call LoadGitHubClient for SettingsCommand (and CacheCommand)?

Reply:
I think that makes more sense. I had to make that change in the program.cs since that is where it is being called. The github client is no longer loaded for the cache/settings command